src/agent.ts

```diff
@@ -0,0 +1,43 @@
+import { Finding, FindingSeverity, FindingType, HandleTransaction, TransactionEvent } from "forta-agent";
+
+export const CREATE_BOT_FUNCTION =
+  "function createAgent(uint256 agentId, address, string calldata metadata, uint256[] calldata chainIds) external";
+export const AGENT_REGISTRY_ADDRESS = "0x61447385b019187daa48e91c55c02af1f1f3f863";
+export const DEPLOYER_ADDRESS = "0x88dC3a2284FA62e0027d6D6B1fCfDd2141a143b8";
+
+const provideHandleTransaction: () => HandleTransaction = () => {
```

Comment:
```suggestion
export function provideHandleTransaction(): HandleTransaction {
```

Values are passed as arguments (not directly) to the handler function by wrapping it inside a `provideHandleTransaction`/`provideHandleBlock` function